dubbo-container/dubbo-container-api/src/main/java/org/apache/dubbo/container/Main.java

```diff
@@ -33,6 +33,8 @@
 
 /**
  * Main. (API, Static, ThreadSafe)
+ *
+ * This class is entry point loading containers.

```

Comment:
Is this saying better? "This class is an entry point to load containers."

Reply:
I think it's fine.